tests/test_cases.md

```diff
@@ -36,11 +36,14 @@ mozdownload --url=https://mozqa.com/index.html -d url-downloads
 mozdownload --url=https://mozqa.com/index.html --username=username --password=password
```

Comment:
It's about the DirectScraper with BasicAuth. We landed tests for all that recently.